userdetails-plugin/grails-app/views/registration/createAccount.gsp

```diff
@@ -370,7 +394,7 @@
 
          $("#setupMFA").click(function(e) {
              $.ajax({
-             url: "${createLink(action:'getSecretForMfa', controller: 'Registration')}",
+             url: "${createLink(uri:'/registration/getSecretForMfa')}",
```

Comment:
Why change from action: controller: which should be stable under refactoring or URL Mappings changes to uri?